modules/extensions-xml-support/src/test/resources/modules/module-json-custom-types-catalog.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<types:mule xmlns="http://www.mulesoft.org/schema/mule/core"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:types="http://www.mulesoft.org/schema/mule/types">
+<catalogs xmlns="http://www.mulesoft.org/schema/mule/types" >
 
-    <types:catalog>
-        <types:type name="Type1" location="./module-json-custom-types-type1-schema.json"/>
-        <types:type name="Type2" location="./module-json-custom-types-type2-schema.json"/>
-    </types:catalog>
+    <catalog name="Type1" format="application/json">
+        <schema format="application/json+schema" location="./module-json-custom-types-type1-schema.json" />
```

Comment:
We have a feature in the runtime that allows to put a file inplace in the config and it should be use the general purpose tool for picking up values from a file. So I would be able to do: <schema format="application/json+schema"> MY_JSON_SCHEMA_HERE </schema> or do:
<schema format="application/json+schema">${file::MY_JSON_SCHEMA_LOCATION_HERE}</schema>